metaflow/metaflow_config.py

```diff
@@ -363,6 +363,31 @@ def _validate_choice(name, value):
 
 KUBERNETES_SANDBOX_INIT_SCRIPT = from_conf("METAFLOW_KUBERNETES_SANDBOX_INIT_SCRIPT")
 
+# Argo Events
```

Comment:
I would put "ARGO" in the name somewhere. This seems a bit too generic given that it is specifically for argo.